src/nnfusion/core/operators/util/annotations.hpp

```diff
@@ -24,23 +25,26 @@ namespace nnfusion
 {
     struct oi_pair
     {
-        oi_pair(size_t out, size_t in, bool destruct)
+        oi_pair(size_t out, size_t in, bool destruct, bool force_inplace = false)
```

Comment:
seems this constructor is needless, we only need to default offset=0 in the other one?